pkg/webservice/dao/event_record.go

```diff
@@ -23,6 +23,7 @@ import (
 )
 
 type EventRecordDAO struct {
+	UUID         string
```

Comment:
Nit: Let's name this something more descriptive (maybe InstanceUUID or something).